BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -565,46 +565,20 @@ - (void)processResponse:(NSURLResponse *)response
         switch (self.HTTPResponse.statusCode)
         {
             case BOXContentSDKAPIErrorAccepted:
-                errorCode = BOXContentSDKAPIErrorAccepted;
-                break;
             case BOXContentSDKAPIErrorBadRequest:
-                errorCode = BOXContentSDKAPIErrorBadRequest;
-                break;
             case BOXContentSDKAPIErrorUnauthorized:
-                errorCode = BOXContentSDKAPIErrorUnauthorized;
-                break;
             case BOXContentSDKAPIErrorForbidden:
-                errorCode = BOXContentSDKAPIErrorForbidden;
-                break;
             case BOXContentSDKAPIErrorNotFound:
-                errorCode = BOXContentSDKAPIErrorNotFound;
-                break;
             case BOXContentSDKAPIErrorMethodNotAllowed:
-                errorCode = BOXContentSDKAPIErrorMethodNotAllowed;
-                break;
             case BOXContentSDKAPIErrorConflict:
-                errorCode = BOXContentSDKAPIErrorConflict;
-                break;
             case BOXContentSDKAPIErrorPreconditionFailed:
-                errorCode = BOXContentSDKAPIErrorPreconditionFailed;
-                break;
             case BOXContentSDKAPIErrorRequestEntityTooLarge:
-                errorCode = BOXContentSDKAPIErrorRequestEntityTooLarge;
-                break;
             case BOXContentSDKAPIErrorPreconditionRequired:
-                errorCode = BOXContentSDKAPIErrorPreconditionRequired;
-                break;
             case BOXContentSDKAPIErrorTooManyRequests:
-                errorCode = BOXContentSDKAPIErrorTooManyRequests;
-                break;
             case BOXContentSDKAPIErrorInternalServerError:
-                errorCode = BOXContentSDKAPIErrorInternalServerError;
-                break;
             case BOXContentSDKAPIErrorInsufficientStorage:
-                errorCode = BOXContentSDKAPIErrorInsufficientStorage;
+                errorCode = self.HTTPResponse.statusCode;
```

Comment:
The previous style... its hard to be certain that each case was really doing the right thing... setting the correct value. Easy to intro a copy/paste error with code like that. The new pattern is way better.

I came across this while testing my change (forcing pseudo 202 responses)